styles/style.css

```diff
@@ -0,0 +1,680 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    font-family: sans-serif;
+    font-size: 1.3rem;
+}
+
+header {
+    display: flex;
+    flex-direction: row;
+    background-image: url(../images/header-image.jpg);
+    background-repeat: no-repeat;
+    background-size: cover; 
+    width: 100%;
+    height: 375px;
+}
+
+.top-left {
+    display: flex;
+    flex-direction: column;
+    padding: 4rem 0 0 8rem;
+}
+
+div.checkout {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    padding: 4rem 0;
+}
+
+.top-logo {
+    max-width: 100%; 
+    width: 100%;
+}
+
+.slogan {
+    font-style: italic;
+    font-size: 1.7rem;
+    color: #a1c349;
+    margin-top: 0;
+    width: fit-content;
+}
+
+.slogan p {
+    margin: 0;
+}
+
+[src="./images/cart-icon.svg"] {

```

Comment:
you could just target this using a selector in the checkout class. That way any image in there gets the sizing. You generally don't want to target specific files.

Reply:
Thanks a lot for letting me know!!
Reviewed!